ChangeTests/ChangeUnitTests.cs

```diff
@@ -0,0 +1,96 @@
+namespace ChangeTests
+{
+    [TestClass]
+    public class ChangeUnitTests
+    {
+        private Dictionary<int, int> availableCoins;
+        MyCash myCash;
+
+
+        [TestInitialize]
+        public void Initialize()
+        {
+            availableCoins = new Dictionary<int, int> {
+            { 100, 10 },
+            { 50, 20 },
+            { 10, 50 },
+            { 5, 100 },
+            { 2, 100 },
+            { 1, 100 }
+            };
+
+            myCash = new MyCash(availableCoins);
+        }
+
+
+        [TestMethod]
+        [ExpectedException(typeof(ArgumentException))]
+        public void Calculate_Negative_value()
+        {
+
```

Comment:
Please cleanup the code everywhere that there is no unwanted blank lines in this example at the beginning of the code block(after {) and in the end (before }). Also remove all not needed comments that don't add value. Somewhere you have // Act. somewhere you don't. These comments are redundant and not helping as well as style is inconsistent.